homeassistant/components/srp_energy/const.py

```diff
@@ -1,12 +1,13 @@
 """Constants for the SRP Energy integration."""
 from datetime import timedelta
 
-SRP_ENERGY_DOMAIN = "srp_energy"
+DOMAIN = "srp_energy"
 DEFAULT_NAME = "SRP Energy"
 
 CONF_IS_TOU = "is_tou"
 
 ATTRIBUTION = "Powered by SRP Energy"
+PHOENIX_TIME_ZONE = "America/Phoenix"
```

Comment:
Is this only used in tests? If so, I would define it _in_ the test suite (in a test, in `conftest.py`, etc.).

Reply:
The api loads data from Phoenix. Not just for tests.